pages/common/gource.md

```diff
@@ -0,0 +1,31 @@
+# gource
+
+> Visualize the history of Git, SVN, Mercurial and Bazaar repositories.
+
+- Run gource in a directory:
```

Comment:
I've tested this with various repositories including `tldr` and gource seems to jump back one level until it finds the `.git` files. So it does not display only a portion of the project when you enter a subdirectory.

Example: Running `gource` in `/tldr/` yields the same result as running it in `/tldr/pages/linux/`.